pkgs/top-level/all-packages.nix

```diff
@@ -12644,8 +12644,12 @@ in
   primesieve = callPackage ../development/libraries/science/math/primesieve { };
 
   prison = callPackage ../development/libraries/prison { };
+  
+  proj = proj_6_1_0;
 
-  proj = callPackage ../development/libraries/proj { };
+  proj_6_1_0 = callPackage ../development/libraries/proj { };
+
+  proj_5_2_0 = callPackage ../development/libraries/proj/5.2.0.nix { };
```

Comment:
Why is version 5.2.0 kept?

Reply:
Because I misread [this](https://nixos.org/nixpkgs/manual/#sec-organisation).  Happy to take it out if you like.

Reply:
@vbgl done.